recipe_sha: 1c7046df027a563d82cfb2af381502a5e31a1b7dbd44745f853cea18eb5fb7ef  # The hash of the recipe's meta.yaml at which this recipe failed to build.
skiplist: true # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
log: |-
  2024-06-04T22:03:45 linux-64 Writing index HTML
  2024-06-04T22:03:45 Subdir: linux-aarch64 Gathering repodata
  2024-06-04T22:03:45 linux-aarch64 Writing pre-patch repodata
  2024-06-04T22:03:45 linux-aarch64 Applying patch instructions
  2024-06-04T22:03:45 linux-aarch64 Writing patched repodata
  2024-06-04T22:03:45 linux-aarch64 Building current_repodata subset
  2024-06-04T22:03:45 linux-aarch64 Writing current_repodata subset
  2024-06-04T22:03:45 linux-aarch64 Writing index HTML
  2024-06-04T22:03:45 Subdir: noarch Gathering repodata
  2024-06-04T22:03:45 noarch Writing pre-patch repodata
  2024-06-04T22:03:45 noarch Applying patch instructions
  2024-06-04T22:03:45 noarch Writing patched repodata
  2024-06-04T22:03:45 noarch Building current_repodata subset
  2024-06-04T22:03:45 noarch Writing current_repodata subset
  2024-06-04T22:03:45 noarch Writing index HTML
  2024-06-04T22:03:46 Subdir: linux-64 Gathering repodata
  2024-06-04T22:03:46 linux-64 Writing pre-patch repodata
  2024-06-04T22:03:46 linux-64 Applying patch instructions
  2024-06-04T22:03:46 linux-64 Writing patched repodata
  2024-06-04T22:03:46 linux-64 Building current_repodata subset
  2024-06-04T22:03:46 linux-64 Writing current_repodata subset
  2024-06-04T22:03:46 linux-64 Writing index HTML
  2024-06-04T22:03:46 Subdir: linux-aarch64 Gathering repodata
  2024-06-04T22:03:46 linux-aarch64 Writing pre-patch repodata
  2024-06-04T22:03:46 linux-aarch64 Applying patch instructions
  2024-06-04T22:03:46 linux-aarch64 Writing patched repodata
  2024-06-04T22:03:46 linux-aarch64 Building current_repodata subset
  2024-06-04T22:03:46 linux-aarch64 Writing current_repodata subset
  2024-06-04T22:03:46 linux-aarch64 Writing index HTML
  2024-06-04T22:03:46 Subdir: noarch Gathering repodata
  2024-06-04T22:03:46 noarch Writing pre-patch repodata
  2024-06-04T22:03:46 noarch Applying patch instructions
  2024-06-04T22:03:46 noarch Writing patched repodata
  2024-06-04T22:03:46 noarch Building current_repodata subset
  2024-06-04T22:03:46 noarch Writing current_repodata subset
  2024-06-04T22:03:46 noarch Writing index HTML
  /opt/conda/lib/python3.10/site-packages/conda_build/cli/main_build.py:501: UserWarning: RECIPE_PATH received is a file (/opt/recipe/meta.yaml).
  It should be a path to a folder.
  Forcing conda-build to use the recipe file.
    warnings.warn(
  WARNING: No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.22
  Adding in variants from internal_defaults
  Adding in variants from /opt/host-conda-bld/conda_build_config_0_-e_conda_build_config.yaml
  Adding in variants from /opt/host-conda-bld/conda_build_config_1_-e_bioconda_utils-conda_build_config.yaml
  Attempting to finalize metadata for peakranger
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Getting pinned dependencies: ...working... done
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Getting pinned dependencies: ...working... done
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Getting pinned dependencies: ...working... done
  BUILD START: ['peakranger-1.18-hdcf5f25_9.tar.bz2']
  Reloading output folder (file:///opt/host-conda-bld): ...working... done
  Solving environment (_h_env): ...working... done

  ## Package Plan ##

    environment location: /opt/conda/conda-bld/peakranger_1717538627246/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl


  The following NEW packages will be INSTALLED:

      _libgcc_mutex:    0.1-conda_forge   conda-forge
      _openmp_mutex:    4.5-2_gnu         conda-forge
      boost-cpp:        1.85.0-h44aadfe_1 conda-forge
      bzip2:            1.0.8-hd590300_5  conda-forge
      icu:              73.2-h59595ed_0   conda-forge
      libboost:         1.85.0-hba137d9_1 conda-forge
      libboost-devel:   1.85.0-h00ab1b0_1 conda-forge
      libboost-headers: 1.85.0-ha770c72_1 conda-forge
      libgcc-ng:        13.2.0-h77fa898_7 conda-forge
      libgomp:          13.2.0-h77fa898_7 conda-forge
      libstdcxx-ng:     13.2.0-hc0a3c3a_7 conda-forge
      libzlib:          1.2.13-h4ab18f5_6 conda-forge
      xz:               5.2.6-h166bdaf_0  conda-forge
      zlib:             1.2.13-h4ab18f5_6 conda-forge
      zstd:             1.5.6-ha6fb4c9_0  conda-forge

  Traceback (most recent call last):
    File "/opt/conda/bin/conda-build", line 11, in <module>
      sys.exit(execute())
    File "/opt/conda/lib/python3.10/site-packages/conda_build/cli/main_build.py", line 590, in execute
      api.build(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/api.py", line 250, in build
      return build_tree(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 3638, in build_tree
      packages_from_this = build(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2409, in build
      create_build_envs(top_level_pkg, notest)
    File "/opt/conda/lib/python3.10/site-packages/conda_build/build.py", line 2183, in create_build_envs
      environ.create_env(
    File "/opt/conda/lib/python3.10/site-packages/conda_build/environ.py", line 1049, in create_env
      _execute_actions(prefix, precs)
    File "/opt/conda/lib/python3.10/site-packages/conda_build/environ.py", line 1332, in _execute_actions
      progressive_fetch_extract.execute()
    File "/opt/conda/lib/python3.10/site-packages/conda/core/package_cache_data.py", line 891, in execute
      raise CondaMultiError(not_cancelled)
  conda.CondaMultiError: Error with archive /opt/conda/pkgs/libboost-headers-1.85.0-ha770c72_1.conda.  You probably need to delete and re-download or re-create this file.  Message was:

  failed with error: [Errno 28] No space left on device
# Last 100 lines of the build log.
